src/main/java/com/spotinst/sdkjava/example/elastigroup/aws/ElastigroupUsageExample.java

```diff
@@ -90,6 +95,9 @@ public static void main(String[] args) throws IOException {
         // Delete subscription (Launch)
         deleteSubscription(subscriptionClient, createdLaunchSubscriptionId);
```

Comment:
deleteSubscription - called twice in this test.

Reply:
fixed